test/e2e/tests/confirmations/alerts/queued-confirmations.spec.ts

```diff
@@ -40,21 +39,25 @@ describe('Queued Confirmations', function () {
             .withSelectedNetworkControllerPerDomain()
             .build(),
           dappOptions: { numberOfDapps: 2 },
-          localNodeOptions: {
-            ...defaultGanacheOptions,
-            concurrent: [
```

Comment:
:information_source:  I've migrated all specs which had the `concurrent` flag already to anvil. This adds more file changes, but it seems better to do it here as we can use the new `localNodeOptions` variable, and we don't need to support this legacy approach at the same time in helpers